content/_data/changelogs/lts.yml

```
@@ -7679,6 +7679,225 @@
 
+- version: "2.361.1"
+  date: 2022-09-07
+  changes:
+
+  - type: major rfe
+    category: major rfe
+    pull: 6083
+    issue: 68570
+    authors:
+      - basil
+      - timja
+      - janfaracik
+      - NotMyFault
+      - jglick
+    pr_title: "[JEP-236] [JENKINS-68570] Require Java 11 or newer"
+    references:
+      - url: "/blog/2022/06/28/require-java-11/"
+        title: Blog post
+      - issue: 68570
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/236/README.adoc
+        title: JEP-236
+      - pull: 6083
+    message: |-
+      Require Java 11 or newer.
+
+  - type: major rfe
+    category: major rfe
+    pull: 6485
+    issue: 68282
+    authors:
+      - janfaracik
+      - NotMyFault
+      - timja
+    pr_title: "[JENKINS-68282] New design for configure project page"
+    message: |-
+      New design for configure project page.
+
+  - type: rfe
+    category: rfe
+    pull: 6585
+    issue: 55582
+    authors:
+      - basil
+      - jglick
+      - timja
+    pr_title: "[JEP-230] [JENKINS-55582] Convert `instance-identity` to a detached\
+      \ plugin"
+    message: |-
+      The <code>instance-identity</code> module has been converted to a detached plugin.
+
+  - type: rfe
+    category: rfe
+    pull: 6873
+    authors:
+      - janfaracik
+    pr_title: Tabs powered by `section-to-tabs.js` appear above its sections,
+      rather than at the very top of the page
+    message: |-
+      Tabs now appear above their respective sections, instead of the top of the page.
+
+  - type: rfe
+    category: rfe
+    pull: 6801
+    issue: 68694
+    references:
+     - url: link:https://github.com/jenkinsci/winstone/releases/tag/winstone-6.1
+       title: Winstone Changelog
+     - url: https://webtide.com/jetty-10-and-11-have-arrived/
+       title: Jetty blog
+     - url: https://github.com/jenkinsci/winstone/pull/232
+       title: Remove ability to load pem cert for winstone
+    authors:
+      - basil
+    pr_title: "Winstone 6.1: upgrade Jetty from 9.4.46.v20220331 to 10.0.11"
+    message: |-
+      Winstone 6.1: Upgrade Jetty from 9.4.46.v20220331 to 10.0.11.
+      Remove support for OpenSSL-style PEM-encoded RSA private keys when running Jenkins with the embedded Jetty container and TLS.
+
+  - type: rfe
+    category: rfe
+    pull: 6772
+    authors:
+      - janfaracik
+      - ridemountainpig
+      - Langer0416
+    pr_title: Modernise the table design
+    references:
+      - issue: 68912
+      - pull: 6760
+      - pull: 6698
+      - issue: 68934
+      - pull: 6687
+      - issue: 68198
+      - pull: 6772
+    message: |-
+      Update to various UI elements and tables modernized.
+
+  - type: rfe
+    category: rfe
+    pull: 6893
+    authors:
+      - janfaracik
+      - timja
+    pr_title: Add a keyboard shortcut to focus the global search bar (CMD + K/CTRL
+      + K)
+    message: |-
+      Keyboard shortcut added to focus global search bar (CMD + K/CTRL + K).
```

```suggestion
      Keyboard shortcut added to focus global search bar (⌘ + K/CTRL + K).
```
Can we use `⌘` here? CMD is nowhere to be found in the UI and automatically translated to `⌘` on Apple devices.

---

Yes, we can definitely add that in, I wasn't sure if the command icon would be appropriate but agree that it would be better.